integration/common/openlineage/common/sql/parser.py

```diff
@@ -7,11 +7,81 @@
 from sqlparse.sql import T, TokenList, Parenthesis, Identifier, IdentifierList
 from sqlparse.tokens import Punctuation
 
-from openlineage.common.models import DbTableName
 
 log = logging.getLogger(__name__)
 
 
+def provider():
+    return "python"
+
+
+class DbTableMeta:
```

Comment:
Putting this class into separate PR would really improve the readability of this one.